package.json

```diff
@@ -1,92 +1,92 @@
 {
-  "name": "prom-client",
```

Comment:
Can you undo the indentation changes in this file please?

Reply:
not easily, its enforced by the prom-client tooling  :-).

Its changed automatically by the eslint config hooks, which husky force-injected. It took me a bit to figure out how to work around this, but I managed to do it by deleting the `.git/hooks/pre-commit` that was put there by husky during `npm install`.

Reply:
Oh man, I'm sorry, I didn't realize that was the cause. You could apply the indentation change in another commit if you want to, otherwise that'll keep happening.

Reply:
no problem, I was puzzled, too! See #318